datumaro/plugins/coco_format/converter.py

```diff
@@ -576,10 +640,12 @@ def apply(self):
                             '' if self._merge_images else subset_name))
                     else:
                         log.debug("Item '%s' has no image info", item.id)
+                segmentation_path = osp.join(self._ann_dir,
```

Comment:
I don't think `merge_images` should be applied to annotations.

I think segmentation path should be made a method in this class to avoid an extra parameter in all the subclasses.

Reply:
Redone, is it ok now?